R/plotting.R

```diff
@@ -84,3 +84,160 @@ plot_avg_rev_benchmark <- function(df_revenue_balance, input_area) {
       values = gss_colour_pallette
     )
 }
+
+# Timeseries Linechart server
+timeseries_linechart_basic <- function(df) {
+  # Long format LA data with tooltip included
+  la_long <- tooltip_func(df)
+  # Build main static plot
+  line_chart <- ggplot2::ggplot(la_long) +
+    ggiraph::geom_line_interactive(
+      ggplot2::aes(
+        x = year,
+        y = average_revenue_balance,
+        color = area_name,
+        data_id = area_name
+      ),
+      na.rm = TRUE,
+      linewidth = 1
+    ) +
+    # Only show point data where line won't appear (NAs)
+    ggplot2::geom_point(
+      data = subset(
+        create_show_point(la_long),
+        show_point
+      ),
+      ggplot2::aes(
+        x = year,
+        y = average_revenue_balance,
+        color = area_name
+      ),
+      shape = 15,
+      size = 1,
+      na.rm = TRUE
+    ) +
+    theme_classic() +
```

Comment:
Switch to AF charts theme:
```suggestion
    afcharts::theme_af() +
```

Reply:
done in larger commit